### DISCUSSION TITLE
ET & BT on control panel, but no ET in artisan?

### DISCUSSION THREAD

**Discussion by Wcrankshaw** [artisan-roaster-scope/artisan#1271] [General]
Hello - First time posting here, and I’ll try to explain myself clearly, but I am certainly not programming savvy. 

I recently purchased an M2 MacBook Pro, and since setting up artisan I no longer have a readout of ET in Artisan. ET and BT probes are both showing readings in the Phidget Control Panel, but only BT is coming across to Artisan.

I’m running a USB VintHub w/ TMP1101 x2 TC. Artisan 2.8.4 and just tried the test version of 2.8.5 with no luck. It previously worked flawlessly on my ancient 2012 Intel MBP. I’ve followed the artisan supplied setup guide and use the preset Phidget machine configuration.

Any thoughts on what’s happening?

**Comment by MAKOMO** [comment]
You are using the latest Phidget driver? Firmware on your TMP1101 and your HUB is current?
What is the ET LCD showing?

Is the Symbolic Assignment "ET Y(x)" under menu Config >> Device, 3rd tab "Symb ET/BT" empty as it should?

**Reply by Wcrankshaw** [reply]
Edit: I was able to test, and that fixed it after removing the Y3 and clicking "update profile." Thank you so much!

That might be it. I can't test on the roaster at the moment, but under ET Y(x) it had "Y3" pre-populated. 

At last roast, ET was displaying "u.u" on the LCD.

Otherwise, yes, everything is up to date as of this morning.
